src/content/changelogs/workers.yaml

```
@@ -10,15 +10,29 @@ entries:
     -  Workers now support the [`allow_custom_ports` compatibility flag](/workers/configuration/compatibility-dates/#allow-specifying-a-custom-port-when-making-a-subrequest-with-the-fetch-api) which enables using the `fetch()` calls to custom ports.
 - publish_date: '2024-07-19'
   description: |-
-    -  Workers with the [mTLS](/workers/runtime-apis/bindings/mtls/) binding now support [Gradual Deployments](/workers/configuration/versions-and-deployments/gradual-deployments/).
+    - Workers with the [mTLS](/workers/runtime-apis/bindings/mtls/) binding now support [Gradual Deployments](/workers/configuration/versions-and-deployments/gradual-deployments/).
+- publish_date: '2024-07-18'
+  description: |-
+    - Added a new `truncated` flag to tail worker events to indicate when the event buffer is full and events are being dropped.
```

```suggestion
    - Added a new `truncated` flag to [Tail Worker](/workers/observability/logging/tail-workers/) events to indicate when the event buffer is full and events are being dropped.
```

Doesn't look like this was documented here?

https://developers.cloudflare.com/workers/observability/logging/tail-workers/

Can we tag author of the change here and make sure we follow through on it? @rohinlohe